Made a card with a video title and text

adiletzainidinov opened the pull request.

adiletzainidinov committed:
Made a card with a video title and text

adiletzainidinov committed:
fixed: add animate for card

Rabiya46 commented:
StyleUsefulVideo.ts так нельзя  писать все стили пишете внутри UsefulVideo.tsx

Rabiya46 commented:
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/f7debab2-f152-4906-b7fd-23c8fc51daa9)


белые пространства остались

Rabiya46 commented:
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/97546046-021b-4e45-b474-49c5343ba0f8)

![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/5733a2b5-6035-4ac4-8fd6-fe7eff32cc37)

вместо иконки ютуба надо заменить на тот, что в фигме 

и вот эти сверху деталики уберите
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/a46b1895-b401-4fdf-a794-5143bddcf385)

Rabiya46 commented:
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/d9ea157c-8187-45ed-842f-2d6c93495955)

box-shadow слишком темный и шрифт Duration не тот 
не так как в фигме

Rabiya46 commented:
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/19f54815-2363-4523-8e23-083d3b39bf1d)

item лучше сделать destruction

Rabiya46 commented:
![image](https://github.com/peaksoft-school/Bilingual-js13/assets/148208912/a430dca0-f46c-4343-af67-8eb47ccff836)

inline style нельзя

Rabiya46 commented on src/pages/LandingPage/components/usefulVideo/UsefulVideo.tsx:
```diff
@@ -0,0 +1,78 @@
+        <ContainerUsefulVideo>
+          <TitleUsefulVideo>Useful videos</TitleUsefulVideo>
+          <ContentUsefulVideo>
+            {CARD_RENDER_DATA.map((item, index) => (
```
item лучше сделать destruction

Rabiya46 commented on src/pages/LandingPage/components/usefulVideo/UsefulVideo.tsx:
```diff
@@ -0,0 +1,78 @@
+                      variant="body2"
+                      component="div"
+                      style={{
+                        color: 'rgba(58, 16, 229, 1)',
```
inline style нельзя

Rabiya46 commented on src/pages/LandingPage/components/usefulVideo/UsefulVideo.tsx:
```diff
@@ -0,0 +1,78 @@
+                    <Typography
+                      variant="body2"
+                      color="text.secondary"
+                      style={{
```
inline style нельзя

Rabiya46 commented on src/pages/LandingPage/components/usefulVideo/usefulVideoData.ts:
```diff
@@ -0,0 +1,23 @@
+    },
+    {
+      third: 3,
+      url: 'https://www.youtube.com/embed/pMPo_huyY4o',
```
поменяйте first, second and third поменяйте на id

Rabiya46 commented on src/pages/LandingPage/components/usefulVideo/UsefulVideo.tsx:
```diff
@@ -0,0 +1,78 @@
+          <ContentUsefulVideo>
+            {CARD_RENDER_DATA.map((item, index) => (
+              <AnimatedCard
+                key={index}
```
если у вас в data есть id лучше дайте id, а не index

adiletzainidinov committed:
fix-bag:
Made the correct structure, removed spaces, icon from figma, destructuring elements, removed inline styles, put id instead of index

adiletzainidinov committed:
fixed: conflict from pacage.json

adiletzainidinov committed:
fixed: conflict pacage

Rabiya46 commented:
![image](https://github.com/user-attachments/assets/c1e6ce5b-73c1-4b47-903e-39ec998d1402)

![image](https://github.com/user-attachments/assets/0aee8796-4952-497d-9056-81d5fc5d1e82)

должно быть так
используйте justify-content, align-items, gap

Rabiya46 commented:
![image](https://github.com/user-attachments/assets/c80f326b-cc72-4217-a8fd-51f47ebb7c7b)

размеры должны быть одинаковыми

Rabiya46 commented on src/assets/react.svg:
delete

Rabiya46 commented on src/pages/MainPage/index.ts:
delete

Rabiya46 commented on src/react-html5video.d.ts:
```diff
@@ -0,0 +1,6 @@
+ 
+    export const DefaultPlayer: ComponentType<any>;
+ }
+ 
```
зачем это?

нельзя any

Rabiya46 commented on src/shared/UI/Input.tsx:
```diff
@@ -0,0 +1,95 @@
+
+const StyledInput = styled(TextField)`
+  width: 500px;
+
```
пишем с одинаковыми стилями

Rabiya46 commented on src/shared/UI/Input.tsx:
```diff
@@ -0,0 +1,95 @@
+    input {
+      color: #23212a;
+    }
+  }
```
пусть если эти цвета есть в theme берите от туда

Rabiya46 commented on src/widgets/usefulVideo/components/UsefulVideo.tsx:
```diff
@@ -0,0 +1,134 @@
+}))
+
+const ContentUsefulVideo = styled('div')`
+   width: 100%;
```
синтаксис разный

Rabiya46 commented on src/widgets/usefulVideo/components/UsefulVideo.tsx:
```diff
@@ -0,0 +1,134 @@
+      fill: rgba(58, 16, 229, 1);
+   }
+`
+//
```
delete

adiletzainidinov committed:
fixed: made @media m same video size,deleted react.svg and index.ts.made same styles,theme didn't have matching colors

adiletzainidinov committed:
fixed: made @media m same video size,deleted react.svg and index.ts.made same styles,theme didn't have matching colors

adiletzainidinov committed:
fixed: conflicts

adiletzainidinov committed:
fixed: conflicts

adiletzainidinov committed:
fixed: conflickts
